pyobjus/pyobjus.pyx

```diff
@@ -68,6 +75,9 @@ class MetaObjcClass(type):
 
     def __new__(meta, classname, bases, classDict):
         meta.resolve_class(classDict)
+        if PY_MAJOR_VERSION == 2:
+            classname = classname.encode("utf-8")
```

Comment:
won't importing unicode_literals from future help here?

Reply:
i wouldn't go that way as it would impact all the others stuff that is fragile